doc/cli.markdown

```diff
@@ -601,9 +601,8 @@ device uuid
 
 show config variables
 
-## env rm &#60;id&#62;
+## env rm ID
 
-Use this command to remove an environment variable from an application
 or device.
```

Comment:
This "or device" line seems to be left on its own. See other comment in the respective source code file.